.github/workflows/mac.yml

```diff
@@ -17,23 +17,19 @@ jobs:
         fetch-depth: 0
 
     - name: Install Qt
-      uses: jurplel/install-qt-action@v3
+      uses: jurplel/install-qt-action@v4
       env:
         ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
       with:
         version: 5.15.2
         target: desktop
         host: mac
-        setup-python: 'false'
+        setup-python: 'true'
         install-deps: true
 
     - name: Print out Qt directory
       run: echo $Qt5_DIR; echo $Qt5_Dir
 
-    - name: Prepare
-      run: |
-        ln -s /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers/X11/ /usr/local/include/X11
-
     - name: make
       run: ./build.sh -DQt_PATH="${Qt5_Dir}"
```

Comment:
Progress! Now the configure is failing, this should be changed to `${Qt_dir}` or path or whichever it is now...